apis/management.cattle.io/v3/k8s_defaults.go

```diff
@@ -346,18 +349,25 @@ var (
 	// ToolsSystemImages default images for alert, pipeline, logging
 	ToolsSystemImages = struct {
 		AlertSystemImages    AlertSystemImages
-		PipelineSystemImages PipelineSystemImages
+		PipelineSystemImages projectv3.PipelineSystemImages
 		LoggingSystemImages  LoggingSystemImages
 	}{
 		AlertSystemImages: AlertSystemImages{
 			AlertManager:       m("prom/alertmanager:v0.11.0"),
 			AlertManagerHelper: m("rancher/alertmanager-helper:v0.0.2"),
 		},
-		PipelineSystemImages: PipelineSystemImages{
-			Jenkins:       m("jenkins/jenkins:2.107-slim"),
-			JenkinsJnlp:   m("jenkins/jnlp-slave:3.10-1-alpine"),
-			AlpineGit:     m("alpine/git:1.0.4"),
-			PluginsDocker: m("plugins/docker:17.12"),
+		PipelineSystemImages: projectv3.PipelineSystemImages{
+			Jenkins:     m("rancher/pipeline-jenkins-server:v0.1.0"),
+			JenkinsJnlp: m("jenkins/jnlp-slave:3.10-1-alpine"),
+			AlpineGit:   m("alpine/git:1.0.4"),
+			//FIXME
```

Comment:
why fix me? do we have to push the image to rancher, or/and create a rancher github repo?

Reply:
yes, we will use the image in rancher registry